src/VisualStudio/Core/Def/Implementation/ProjectSystem/VisualStudioAnalyzer.cs

```diff
@@ -149,6 +149,9 @@ public override string FullPath
             public override object Id
                 => _underlying.Id;
 
+            public override string Display
+                => _underlying.Display;
```

Comment:
❓ Why is this not using the same form as `UnresolvedAnalyzerReference.Display`?

https://github.com/dotnet/roslyn/blob/97ad88fd2e1a97e15ccd5e9c05b096888ded8a60/src/Compilers/Core/Portable/DiagnosticAnalyzer/UnresolvedAnalyzerReference.cs#L28-L34

Reply:
I think that is what this change will produce given `_underlying` is of type `UnresolvedAnalyzerReference`